### PR TITLE
#920 add slash when eureka.client.service-url.defaultZone does not contain one

### PR DESCRIPTION
**Pull request by sfat**

### CODE REVIEW

**Comment by spencergibb** [comment]
Have you signed the [Contributor License Agreement](https://support.springsource.com/spring_committer_signup)?

File: spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/EurekaClientConfigBean.java [hunk 1]
```diff
@@ -430,13 +431,25 @@ public String fetchRegistryForRemoteRegions() {
 		if (serviceUrls == null || serviceUrls.isEmpty()) {
 			serviceUrls = this.serviceUrl.get(DEFAULT_ZONE);
 		}
-		if (serviceUrls != null) {
-			return Arrays.asList(serviceUrls.split(","));
+		if (!StringUtils.isEmpty(serviceUrls)) {
+			final String[] serviceUrlsSplit = serviceUrls.split(",");
```
Comment (spencergibb) [inline spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/EurekaClientConfigBean.java]: We should probably use `StringUtils.commaDelimitedListToStringArray` here.

**Comment by spencergibb** [comment]
Nice tests!

**Comment by sfat** [comment]
@spencergibb yes, I have signed the aggreement.

File: spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/EurekaClientConfigBean.java (hunk 1 above)
Comment (sfat) [inline spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/EurekaClientConfigBean.java]: sure

**Pull request closed without merging**